docs/source/guides/logging.rst

```diff
@@ -3,4 +3,83 @@
 Logging
 =======
 
-TODO
+BentoML provides a powerful and detailed logging pattern out of the box. Request logs for webservices are logged along with requests to each of the model runner services. We use `RichHandler <https://rich.readthedocs.io/en/stable/logging.html>`_ to provide appropriate color coding to the logs in order to make them more easily read.
+
+.. image:: ../_static/img/bentoml-logs.png
+
+The log format is as follows:
+
+.. parsed-literal::
+
+    "[%(component)s] %(message)s"
```

Comment:
I think this format is less relevant for users. For access logs, we can breakdown a log entry.
```
[api_server] 127.0.0.1:55723 (scheme=http,method=POST,path=/classify,type=application/json,length=9) (status=200,type=application/json,length=1) 0.465ms (trace=175923203261911804790364073480797463970,span=2163644694959317809,sampled=0)
```
In the access log example, the format has the following components.
- Component
- Client IP
- Request
- Response
- Latency
- Traces

Reply:
👍